lib/services/storage_service.ts

```diff
@@ -203,12 +203,14 @@ export class SNStorageService extends PureService {
   }
 
   /** @todo This function should be debounced. */
-  private async repersistToDisk() {
-    if (!this.storagePersistable) {
-      return;
-    }
-    if (this.persistencePolicy === StoragePersistencePolicies.Ephemeral) {
-      return;
+  private async repersistToDisk(force = false) {
+    if (!force) {
+      if (!this.storagePersistable) {
+        return;
+      }
+      if (this.persistencePolicy === StoragePersistencePolicies.Ephemeral) {
```

Comment:
Should `force` encapsulate the `persistencePolicy` as well? I'd say no. If ephemeral we should never write to storage. In this case the less power `force` has the better. At least until we explicitly need this power. For now we only need it to override `storagePersistable`.

You should definitely document/comment somewhere why we might need to use force here.